src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -71,63 +114,115 @@ ProtoMulticastCommunicationProfile_t *primitive_msg_listener_profile;
 static TbotsProto_Vision vision_msg;
 static TbotsProto_RobotStatus robot_status_msg;
 static TbotsProto_RobotLog robot_log_msg;
-static TbotsProto_Primitive primitive_msg;
+static TbotsProto_PrimitiveSet primitive_set_msg;
 
 /* USER CODE END Variables */
 /* Definitions for NetStartTask */
 osThreadId_t NetStartTaskHandle;
+uint32_t NetStartTaskBuffer[1024];
```

Comment:
would it be better to make 1024 a constant

Reply:
this is auto generated, can't change :(